app/src/main/java/org/wikipedia/edit/richtext/ColorSpanEx.kt

```diff
@@ -0,0 +1,16 @@
+package org.wikipedia.edit.richtext
+
+import android.text.TextPaint
+import android.text.style.ForegroundColorSpan
+import androidx.annotation.ColorInt
+
+class ColorSpanEx(@ColorInt foreColor: Int, @field:ColorInt @param:ColorInt private val backColor: Int,
```

Comment:
Do you know the reason why this `backColor` needs two annotations instead of one `@ColorInt`?

Reply:
nope!